CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [4.6.1] - 2023-09-28
```

Comment:
let's keep this changelog diff 👍🏻 but no need to upgrade the version in features PR (releases and version bump must come from main), maybe more stuff is coming into develop :wink:

Reply:
No problem with that but discipline must be enforced: changes that these PR further change went into main without ever being recorded in the changelog (and that was an API change)